src/pages/menuPage/MenuPage.js

```diff
@@ -0,0 +1,97 @@
+import React,{useState} from 'react';
+import Navbar from '../../components/navbar/Navbar';
+import Navlink from '../../components/menu-page-navlinks/Navlink'
+import friedRice from '../../components/reusableMenuCard/dishImages/friedRice.png'
+import jollofRice from '../../components/reusableMenuCard/dishImages/jollofRice.png'
+import egusiSoup from '../../components/reusableMenuCard/dishImages/egusiSoup.png'
+import oraSoup from '../../components/reusableMenuCard/dishImages/oraSoup.png'
+import whiteRice from '../../components/reusableMenuCard/dishImages/whiteRice.png'
+import okroSoup from '../../components/reusableMenuCard/dishImages/okroSoup.png'
+import Dish from '../../components/reusableMenuCard/Dish'
+import './MenuPage.css';
+import Footer from '../../components/footer/Footer';
+
+const MenuPage = () => {
+    const [list, setList] = useState([
+        {
+            image: friedRice,
+            name: 'Fried Rice',
+            price: '₦1000',
+            description: 'Fried rice with chicken laps'
+        },
+        {
+            image: jollofRice,
+            name: 'Jellof Rice',
+            price: '₦800',
+            description: 'Nigerian party jellof'
+        },
+        {
+            image: egusiSoup,
+            name: 'Egusi Soup',
+            price: '₦1500',
+            description: 'Egusi Soup with plenty meat'
+        },
+        {
+            image: oraSoup,
+            name: 'Ora Soup',
+            price: '₦1500',
+            description: 'Ora soup with fish and two beef and semo'
+        },
+        {
+            image: whiteRice,
+            name: 'White Rice',
+            price: '₦1200',
+            description: 'White rice and stew with chicken'
+        },
+        {
+            image: okroSoup,
+            name: 'Okro Soup',
+            price: '₦1000',
+            description: 'Okro soup with plenty meat and fish'
+        }
+    ])
+
+    return (
+        <div className='menu'>
+            <Navbar />
+            <section className='menu-header'>
+                <h1>
+                    Test the goodness of fresh and healthy foods
+                </h1>
+            </section>
+            <section >
+                < Navlink />
```

Comment:
This is not supposed to be a navbar or navlink.... It is suppose to be a tab link and you are not to use react router link to navigate, rather you are meant to use local state to switch veiw/data in this page